src/autosplitters/supermetroid.rs

```diff
@@ -2233,10 +2233,10 @@ impl SNESState {
     ) -> Result<SNESSummary, Box<dyn Error>> {
         let start_time = Instant::now();
         let snes_data = client.get_addresses(&[
-            (0xF5008B, 2),
-            (0xF5079B, 3),
-            (0xF50998, 1),
-            (0xF509A4, 61),
+            (0xF5008B, 2),  // Controller 1 Input
```

Comment:
Thanks for documenting this. I really should have made a comment like this myself.